src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +94,57 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files and adds metadata to overview-features page
+     *
+     * @param propertiesFiles
+     *            property files to read
+     */
+    public void parsePropertiesFiles(List<String> propertiesFiles) {
+
+        if (propertiesFiles.isEmpty()) {
+
+            return;
+
+        }
+
+        for (String propertyFile : propertiesFiles) {
+
+            LOG.info("Processing MetadataFile - %s", propertyFile);
+
+            processMetaDataFile(propertyFile);
+
+        }
+
+    }
+
+    private void processMetaDataFile(String file){
+
+        try {
+
+            PropertiesConfiguration config = new PropertiesConfiguration(file);
+
+            Iterator<String> keys = config.getKeys();
+
+            while (keys.hasNext()) {
+
+                String key = keys.next();
+
+                String value = config.getProperty(key).toString();
+
+                LOG.info("Adding classification - %s%s sourced from properties file - %s", key, value, file);
```

Comment:
this is what you see when the report is generated so my question is - is it valuable?

Reply:
makes sense will remove it...